Sources/App/SlackCommand+Options.swift

```diff
@@ -6,13 +6,17 @@ extension SlackCommand {
     enum Option {
         static let branch = "branch"
         static let repo = "repo"
+        static let version = "version"
```

Comment:
I'd either use `enum Option: String` with `case` here, or even better, use:
```swift
struct Option: ExpressibleByStringLiteral {
    let name: String
    init(stringLiteral value: String) {
        name = value
    }

    static let branch: Option = "branch"
    static let repo: Option = "repo"
    static let version: Option = "version"
}
```

That way we will be able to call `metadata.value(forOption: .branch)`

Reply:
👍